main.py

```diff
@@ -232,9 +312,10 @@ def escape_path(path):
 		accelerator_arg = Xqemu.generateAcceleratorArg(settings.settings['use_accelerator'])
 
 		dvd_path_arg = ''
-		if settings.settings['dvd_present']:
-			check_path(settings.settings['dvd_path'])
-			dvd_path_arg = ',file=' + escape_path(settings.settings['dvd_path'])
+		if main.tableGames.currentRow() > 0:
+			selectedGame = main.gamesCache[main.tableGames.currentRow() - 1]
```

Comment:
I believe there's a better way to get internal / user-data which belong to a table row / cell.
Did you look for cleaner alternatives?